keras_applications/mobilenet_v2.py

```diff
@@ -161,6 +167,27 @@ def MobileNetV2(input_shape=None,
                  are used at each layer.
         depth_multiplier: depth multiplier for depthwise convolution
             (also called the resolution multiplier)
+        min_layer_filters: The minimum number of filters to use in any layer.
```

Comment:
"min_filters" is a shorter name. Please make the description clearer, it isn't entirely clear how this argument works, from the description.